concent_api/core/views.py

```diff
@@ -273,7 +353,9 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
             client_message.TYPE,
             client_message.report_computed_task.task_to_compute.compute_task_def['task_id'],
             client_message.serialize(),
-            status = ReceiveStatus,
+            provider_public_key  = other_party_public_key,
```

Comment:
You're not checking if `other_party_public_key` is not `None`.

Reply:
base64.b64decode can't return None.

Reply:
But it does not come directly from `b64decode()`. It comes from `decode_other_party_public_key()`.

Reply:
I see that I misread the conditions in `decode_other_party_public_key()`. In that case it's OK.